molecule/driver/vagrantdriver.py

```diff
@@ -22,7 +22,10 @@
 import copy
 import os
 
-import vagrant
+try:
+    import vagrant
+except ImportError:
+    LOG.error('Driver missing, install python-vagrant!')
```

Comment:
LOG hasn't been imported at this point so this fails. In hindsight I think what we are really going to want is something like this.

```
try:
    import vagrant
except ImportError:
    sys.exit('ERROR: Driver missing, install python-vagrant!')
```

This way we don't require the LOG import and the program exits with a non-zero status.